sonarts-sq-plugin/sonar-typescript-plugin/src/main/java/org/sonar/plugin/typescript/ExternalTypescriptSensor.java

```diff
@@ -185,8 +186,9 @@ private static void checkCompatibleNodeVersion(String nodeExecutable) {
     for (InputFile inputFile : inputFiles) {
       File tsConfig = findTsConfig(inputFile, projectBaseDir);
       if (tsConfig == null) {
-        String message = "No tsconfig.json file found for [%s] (looking up the directories tree until project base directory [%s]). This file will not be analyzed.";
-        LOG.error(String.format(message, inputFile.uri(), projectBaseDir.getAbsolutePath()));
+        String message = "No tsconfig.json file found for [%s] (looking up the directories tree until project base directory [%s]). Using default configuration.";
+        LOG.warn(String.format(message, inputFile.uri(), projectBaseDir.getAbsolutePath()));
+        inputFileByTsconfig.computeIfAbsent(DEFAULT_TSCONFIG, x -> new ArrayList<>()).add(inputFile);
       } else {
         inputFileByTsconfig.computeIfAbsent(tsConfig.getAbsolutePath(), x -> new ArrayList<>()).add(inputFile);
       }
```

Comment:
log info: No tsconfig.json file found for N files (Run in debug mode to see all of them). They will be analyzed with a default configuration.